common/translation.cpp

```diff
@@ -450,6 +454,55 @@ bool TranslationManager::checkHeader(File &in) {
 	return true;
 }
 
+#ifdef USE_FRIBIDI
+String TranslationManager::bidiAlgo(const String input) {
+	if (getCurrentLanguage() != "he")		//TODO: modify when we'll support other RTL languages, such as Arabic and Farsi
+		return input;
+
+	if (getCurrentCharset() != "iso-8859-8") {
+		warning("Unexpected charset is used with %s language: %s", getCurrentLanguage().c_str(), getCurrentCharset().c_str());
```

Comment:
It is better to put name of the method as prefix to the warning, so it would be easier to find it.

Reply:
Done